### PR TITLE
nohup, stat: add Russian translation

### PR DESCRIPTION
**Pull request by iridacea**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.ru/common/nohup.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# nohup
+
+> Позволяет процессу продолжать работу после закрытия терминала.
+> Больше информации: <https://www.gnu.org/software/coreutils/nohup>.
+
+- Запустить процесс, который может выполняться в отвязке от терминала:
+
+`nohup {{команда}} {{аргументы_команды}}`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/nohup.md:8]: The following change could be applied after the original [page](https://github.com/tldr-pages/tldr/blob/main/pages/common/nohup.md) is updated:

```suggestion
`nohup {{команда}} {{аргумент1 аргумент2 ...}}`
```

Wrong syntax is used for the placeholder supporting zero or more items.

File: pages.ru/common/nohup.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Запустить `nohup` в фоновом режиме:
+
+`nohup {{команда}} {{аргументы_команды}} &`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/nohup.md:12]: The following change could be applied after the original [page](https://github.com/tldr-pages/tldr/blob/main/pages/common/nohup.md) is updated:

```suggestion
`nohup {{команда}} {{аргумент1 аргумент2 ...}} &`
```

Wrong syntax is used for the placeholder supporting zero or more items.

File: pages.ru/common/nohup.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Запустить скрипт оболочки, который может выполняться в отвязке от терминала:
+
+`nohup {{путь/до/скрипта.sh}} &`
+
+- Запустить процесс и перенаправить его вывод в указанный файл:
+
+`nohup {{команда}} {{аргументы_команды}} > {{путь/до/выходного_файла}} &`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/nohup.md:20]: The following change could be applied after the original [page](https://github.com/tldr-pages/tldr/blob/main/pages/common/nohup.md) is updated:

```suggestion
`nohup {{команда}} {{аргумент1 аргумент2 ...}} > {{путь/до/выходного_файла}} &`
```
Wrong syntax is used for the placeholder supporting zero or more items.

**Review by EmilySeville7cfg — APPROVED** [review]
No changes are requested because translations are already synced with the original pages requering fixes.

**Review by navarroaxel — APPROVED** [review]

File: pages.ru/common/nohup.md (hunk 1 above)
Comment (iridacea) [reply pages.ru/common/nohup.md:8]: This and two other suggestions are implemented in #9464

**Pull request merged**